src/modules/views/Folder/virtualized/Table.jsx

```diff
@@ -15,9 +15,14 @@ import { useNewItemHighlightContext } from '@/modules/upload/NewItemHighlightPro
 const TableRow = forwardRef(({ item, context, children, ...props }, ref) => {
   const { isItemCut } = useClipboardContext()
   const isCut = isItemCut(item._id)
+  const { actions } = context
 
   return (
-    <RightClickFileMenu doc={item} actions={context.actions} {...props}>
+    <RightClickFileMenu
+      doc={item}
+      actions={actions.filter(action => !action.selectAllItems)}
```

Comment:
This is not the right way to do it; check how the `versions` action does it ( it does not display when having multiple files ) you can do the other way around.